### PR TITLE
refactor: remove cloneDeep

### PR DESCRIPTION
**Pull request by luhc228**

### CODE REVIEW

**Comment by wssgcg1213** [comment]
check the CI aha~

File: packages/plugin-rax-compat/src/index.ts [hunk 1]
```diff
@@ -100,10 +100,10 @@ const plugin: Plugin<CompatRaxOptions> = (options = {}) => ({
       // TODO: optimize the logic, support deep merge in plugin API.
       // Must create a variable to store the original compilationConfig.
       const originalSwcCompilationConfig = typeof config.swcOptions?.compilationConfig === 'object'
-        ? cloneDeep(config.swcOptions?.compilationConfig || {})
+        ? cloneDeep(config.swcOptions.compilationConfig)
         : {};
-      const compilationConfigFunc = typeof config.swcOptions?.compilationConfig === 'function'
-        ? config.swcOptions?.compilationConfig
+      const originalSwcCompilationConfigFunc = typeof config.swcOptions?.compilationConfig === 'function'
+        ? cloneDeep(config.swcOptions.compilationConfig)
```
Comment (ClarkXia) [inline packages/plugin-rax-compat/src/index.ts:106]: type function 为什么要 cloneDeep？

Comment (luhc228) [reply packages/plugin-rax-compat/src/index.ts]: Removed.

**Comment by ClarkXia** [comment]
changelog is required.

**Review by ClarkXia — APPROVED** [review]

**Review by linbudu599 — APPROVED** [review]

**Review by ClarkXia — APPROVED** [review]

**Review by linbudu599 — APPROVED** [review]

**Pull request merged**